### DISCUSSION TITLE
question: is north always -y and east +x

### DISCUSSION THREAD

**Discussion by tom-twinlabs** [RodrigoHamuy/react-three-map#113] [General]
This is what it looks like to me, by adding AxesHelper, I just wanted to ask if this is always the case? 

Many thanks

**Comment by RodrigoHamuy** [comment]
I was considering letting the user pick the camera UP vector, so you can decide what North East are, but at the moment is kinda following defaults threejs behaviour, where Y is UP and X is right and -Z forward.

So north is -z and east is +x.
```
        N
       - z

W - x   ⨁   + x E

       + z
        S
```

![image](https://github.com/RodrigoHamuy/react-three-map/assets/2405611/f4b6dee8-37ea-4144-865c-a19680f83bbb)


It shouldn't be hard to have +Y for north instead, and would make more sense for a lot of cases. PRs are welcomed for that feature 😄

**Comment by tom-twinlabs** [comment]
Thank you for the response!

I actually don't have a need to change camera.up at the moment. Aligning with default three.js behavior makes total sense. I will make a small PR for a docs addition, but feel free to ignore it :)

**Reply by tom-twinlabs** [reply]
PR here https://github.com/RodrigoHamuy/react-three-map/pull/119
